src/partials/languages/it.html

```diff
@@ -41,5 +41,22 @@
   "source.revision.date": "Ultimo aggiornamento",
   "source.file.date.updated": "Ultimo aggiornamento",
   "source.file.date.created": "Creata",
-  "toc.title": "Indice"
+  "toc.title": "Indice",
+  "direction": "ltr",
```

Comment:
Could you please put the keys in the same order as in the `en.html`?

Reply:
Sure. Thank you

https://github.com/squidfunk/mkdocs-material/blob/c67f6007de983764edfafaa6b0b908af68986a21/src/partials/languages/it.html